tools/ansible/roles/dockerfile/templates/supervisor.conf.j2

```diff
@@ -74,8 +74,19 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
+[program:awx-cache-clear]
+command = awx-manage run_cache_clear
+autorestart = true
+startsecs = 30
+stopasgroup=true
+killasgroup=true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
 [group:tower-processes]
-programs=nginx,uwsgi,daphne
+programs=nginx,uwsgi,daphne,wsbroadcast,awx-cache-clear
```

Comment:
`wsbroadcast` exists in the devel branch but not in this branch or the feature branch. I'm thinking this is an artifact of a git conflict.

On the other hand... this file does contain a `heartbeet` that isn't in devel, which should be listed here.

Given the current implementation, I don't think that this PR necessarily needs to make any changes to this file, because it runs the dispatcher and that will run the task itself.

Reply:
wsbroadcast and heartbeet might be better added in the feature branch since they're missing there and not relevant to this PR.